client/js/models/AboutUs.js

```diff
@@ -0,0 +1,9 @@
+module.exports = { ...require('./__proto__.js'),
+	data: [
+		{class: 'arrow', filename: 'LeftArrow.png', alt: 'Left Arrow', date: '', caption: ''},
```

Comment:
Use `klass` instead of `class` in order to avoid clashes with the reserved js word `class`. You're namespaced in an object here, so it works fine, but still best to use `klass`.